### PR TITLE
Added showGlobalVariables as a workspace level parameter

### PR DESCRIPTION
**Pull request by aschade92**
Before, this option was only available in `launch.json` but this PR makes `showGlobalVariables` as a workspace level setting so that things like the debug and run test actions can use this setting.

### CODE REVIEW

**Comment by aschade92** [comment]
https://github.com/Microsoft/vscode-go/issues/2323

File: package.json [hunk 1]
```diff
@@ -1233,7 +1238,8 @@
                 "maxVariableRecurse": 1,
                 "maxStringLen": 64,
                 "maxArrayValues": 64,
-                "maxStructFields": -1
+                "maxStructFields": -1,
+                "showGlobalVariables": false
```
Comment (ramya-rao-a) [inline package.json]: I believe we need to pull `showGlobalVariables` one level up.
The properties in LoadConfig is a 1:1 to what is at https://github.com/go-delve/delve/blob/4c9a72e486f1f0d0c90ecede8415a871dced8117/service/api/conversions.go#L276

**Comment by aschade92** [comment]
@ramya-rao-a updated

**Comment by ramya-rao-a** [comment]
@jhendrixMSFT FYI, I am changing the default value for `showGlobalVariables` to `false` as it is still causing perf issues for some users and the benefits of having it enabled doesnt outweigh the issues.

**Comment by ramya-rao-a** [comment]
@aschade92 Looks like this is your first PR contribution to this project, Thanks & Welcome!

I have pushed 2 more commits. 1 to update the description and default value for the `dlvLoadConfig` setting, the other is explained at https://github.com/Microsoft/vscode-go/pull/2351#issuecomment-468959010

**Comment by aschade92** [comment]
@ramya-rao-a sounds good, thanks for reviewing!

**Pull request merged**